nvflare/lighter/provision.py

```diff
@@ -86,6 +112,27 @@ def main():
     participants = list()
     for p in project_dict.get("participants"):
         participants.append(Participant(**p))
+    if args.add_user:
+        try:
+            extra = load_yaml(os.path.join(current_path, args.add_user))
+            extra.update({"type": "admin"})
+            participants.append(Participant(**extra))
+        except BaseException:
+            print("** Error during adding user **")
+            print("The yaml file format is")
+            print(adding_user_error_msg)
+            exit(0)
+    if args.add_client:
+        try:
+            extra = load_yaml(os.path.join(current_path, args.add_client))
+            extra.update({"type": "client"})
+            participants.append(Participant(**extra))
+        except BaseException as e:
+            print("Error during adding client")
```

Comment:
this message is not consistent with "** Error during adding user **"

suggest unify their style.

Reply:
Thanks.  Fixed.